src/application/controllers/tasks/load-current-month-tasks-controller.ts

```diff
@@ -24,13 +24,22 @@ export class LoadCurrentMonthTasksController implements Controller {
         return badRequest(error)
       }
 
-      const { tagId } = httpRequest.query
+      const { tagId, page = 1, limit = 10 } = httpRequest.query
 
       const now = new Date()
       now.setUTCHours(0, 0, 0, 0)
 
-      const result = await this.loadCurrentMonthTasks.load({ date: now, tagId })
-      return success(result)
+      const offset = (Number(page) - 1) * Number(limit)
+
+      const result = await this.loadCurrentMonthTasks.load({ date: now, tagId, limit: Number(limit), offset })
+      return success({
+        data: result,
+        pagination: {
+          page: Number(page),
+          limit: Number(limit),
+          count: result.length
+        }
+      })
```

Comment:
Acho que fica melhor colocar assim:
```suggestion
      return success({
        data: result,
        page: Number(page),
        limit: Number(limit),
        count: result.length
      })
```
E o parse de `page` e `limit` pode ser feito antes já que é usado no `this.loadCurrentMonthTasks.load` e no retorno